src/Microsoft.ML.StandardLearners/Standard/LinearClassificationTrainer.cs

```diff
@@ -222,21 +222,26 @@ internal virtual void Check(IHostEnvironment env)
             }
         }
 
-        internal const string SDCADetailedSummary = @"This classifier is a trainer based on the Stochastic DualCoordinate 
-Ascent(SDCA) method, a state-of-the-art optimization technique for convex objective functions.
+        internal const string Remarks = @"<remarks>
+This classifier is a trainer based on the Stochastic DualCoordinate Ascent(SDCA) method, a state-of-the-art optimization technique for convex objective functions.
 The algorithm can be scaled for use on large out-of-memory data sets due to a semi-asynchronized implementation 
 that supports multi-threading.
+<para>
 Convergence is underwritten by periodically enforcing synchronization between primal and dual updates in a separate thread.
 Several choices of loss functions are also provided.
 The SDCA method combines several of the best properties and capabilities of logistic regression and SVM algorithms.
-For more information on SDCA, see:
-<see href='https://www.microsoft.com/en-us/research/wp-content/uploads/2016/06/main-3.pdf'>Scaling Up Stochastic Dual Coordinate Ascent</see>.
-<see href='http://www.jmlr.org/papers/volume14/shalev-shwartz13a/shalev-shwartz13a.pdf'>Stochastic Dual Coordinate Ascent Methods for Regularized Loss Minimization</see>.
+</para>
+<para>
 Note that SDCA is a stochastic and streaming optimization algorithm. 
-The results depends on the order of the training data. For reproducible results, it is recommended that one sets `shuffle` to
-`False` and `NumThreads` to `1`.
-Elastic net regularization can be specified by the l2_weight and l1_weight parameters. Note that the l2_weight has an effect on the rate of convergence. 
-In general, the larger the l2_weight, the faster SDCA converges.";
+The results depends on the order of the training data. For reproducible results, it is recommended that one sets <paramref>shuffle</paramref> to
+False and <paramref>NumThreads</paramref> to 1.
```

Comment:
><paramref>NumThreads</paramref> [](start = 10, length = 31)

Is the correct form not `<paramref href=`?

Also this seems like kind of a strange solution...

Reply:
it's actually <paramref name="name"/> . Thanks for spotting it. 

---
In reply to: [199549798](https://github.com/dotnet/machinelearning/pull/455#discussion_r199549798) [](ancestors = 199549798)